### DISCUSSION TITLE
0.12.0 (Babelfish)

### DISCUSSION THREAD

**Discussion by Mattk70** [Mattk70/Chirpity-Electron#32] [Announcements]
- Birds' common names can be set to display in any of 30 Languages
- Mac version has application update notifications
- Shortcuts for the Mac version can be accessed with the command key

Bug fixes
- Fixed an issue where switching models after analysing compressed files (mp3, m4a, etc.) would lead to inaccurate predictions
- Fixed an issue with experimental audio filters. Now adjusting these filter settings will activate the filters if they were previously disabled. 
- Command-Q on Mac will now only quit the app if the window is in focus!
- In the settings menu, the inline help link explains that SNR and context modes are not available in BirdNET, which is why these settings are disabled.

<hr /><em>This discussion was created from the release <a href='https://github.com/Mattk70/Chirpity-Electron/releases/tag/v0.12.0'>0.12.0</a>.</em>

**Comment by Vollpflock** [comment]
YOU ARE AWESOME!
